src/esm_runscripts/prepare.py

```diff
@@ -494,6 +493,12 @@ def _add_all_folders(config):
     config["general"]["reusable_filetypes"] = config["general"].get(
         "reusable_filetypes", ["bin", "src"]
     )
+    # Define the files that could be reusable accross runs (external files)
+    config["general"]["potentially_reusable_files"] = (
```

Comment:
I think we should here be consistent and always use `filetypes`. From my understanding of the functionality, you aren't able to change on particular file to update, but an entire category.

Reply:
OK, done, I actually changed it in the other file and forgot ti here.

Reply:
Perfect, thanks!